backend/src/main/java/com/zeros/backend/service/Restaurant.java

```diff
@@ -8,12 +8,15 @@
     private Location location;
     private float rating;
     private List<String> photosReference;
+    private Route route;
 
-    Restaurant(String name, Location location, float rating, List<String> photos) {
+    Restaurant(String name, Location location, float rating, List<String> photos, 
```

Comment:
I'm kind of confused about the notion of restaurant in our object model. the fact that the same restaurant might have two different instances in two different search results is a little bit confusing... we can think of it in a future refacto

Reply:
Ok, then shouldn't we name it this way ?